lib/light-service/organizer/with_reducer.rb

```diff
@@ -36,6 +36,7 @@ def reduce(*actions)
           begin
             invoke_action(current_context, action)
           rescue FailWithRollbackError
+            context[:_rollback] = true
```

Comment:
This is bad. But I did not have a better idea to signal for the previous iterator to roll back. I can't reraise the `FailWithRollbackError` as that is internal and what if something won't handle it. I am open to suggestions on how to make this better.